src/client.js

```diff
@@ -71,8 +107,8 @@ class RPCClient extends EventEmitter {
     this.fetch.endpoint = 'https://discord.com/api';
 
     /**
-     * Raw transport userd
-     * @type {RPCTransport}
+     * The transport used
+     * @type {IPCTransport | WebSocketTransport}
```

Comment:
remove the type change here.

Reply:
i didnt see a RPCTransport typedef anywhere? should i create a typedef for it?

Reply:
typedef created in `ca001fb`